app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :events, foreign_key: :creator_id, class_name: 'Event'
```

Comment:
There is no need of adding `class_name: 'Event'`. Rails is able to process this.

Reply:
Thanks. Issue solved.